src/gui/src/gui.cpp

```diff
@@ -1089,6 +1084,8 @@ int startGui(int& argc,
         // init remainder of GUI, to be called immediately after OpenRoad is
         // guaranteed to be initialized.
         main_window->init(open_road->getSta());
+        // announce design created to ensure GUI gets setup
+        open_road->designCreated();
```

Comment:
Why is this part of statup?

Reply:
I moved this from the script widget to here. It's needed because when loading the gui after loading the design, the gui still needed to be told that a design was there. Maybe the correct thing would be to call main_window to set the Db like previously, but not trigger the designLoaded for everything?